treeview/js/main.js

```diff
@@ -558,6 +560,14 @@
                 function getTreeData(queryConfig) {
                     var treeHeaders = {};
                     treeHeaders[ERMrest.contextHeaderName] = getHeader("main", queryConfig.tree_schema_table);
+                    if(urlParams['pcid'] || urlParams['ppid']){
+                        if(urlParams['pcid'])
+                            header['pcid']=urlParams['pcid']
+                        if(urlParams['ppid'])
+                            header['ppid']=urlParams['ppid']
+                        treeHeaders[ERMrest.contextHeaderName] =header;
+                    }   
```

Comment:
This extra conditional seems unnecessary. Change this to:
```
var header = getHeader("main", queryConfig.tree_schema_table);
if(urlParams['pcid'])
  header['pcid']=urlParams['pcid']
if(urlParams['ppid'])
  header['ppid']=urlParams['ppid']

treeHeaders[ERMrest.contextHeaderName] = ERMrest._certifyContextHeader(header);
```

Reply:
removed conditional statement